cmd/sops/main.go

```diff
@@ -104,6 +105,48 @@ func main() {
    For more information, see the README at github.com/mozilla/sops`
 	app.EnableBashCompletion = true
 	app.Commands = []cli.Command{
+		{
+			Name:      "publish",
+			Usage:     "Publish sops file to a configured destination",
+			ArgsUsage: `file`,
+			Flags: append([]cli.Flag{
+				cli.BoolFlag{
+					Name:  "yes, y",
+					Usage: `pre-approve all changes and run non-interactively`,
+				},
+				cli.BoolFlag{
+					Name:  "verbose",
```

Comment:
isn't this already covered by top-level flags?

Reply:
Yes-ish, with a slight change we can support `sops --verbose publish file`, but we lose support for `sops publish --verbose`. Making a change to support both.